src/platform/nrfconnect/wifi/WiFiManager.cpp

```diff
@@ -469,6 +480,19 @@ void WiFiManager::DisconnectHandler(Platform::UniquePtr<uint8_t>)
     });
 }
 
+void WiFiManager::IPv6PrefixAddHandler(const void * data)
+{
+    // This is needed to send mDNS queries containing new IPv6 prefix
+    chip::DeviceLayer::ChipDeviceEvent event;
+    event.Type = chip::DeviceLayer::DeviceEventType::kDnssdRestartNeeded;
+
+    CHIP_ERROR error = chip::DeviceLayer::PlatformMgr().PostEvent(&event);
+    if (error != CHIP_NO_ERROR)
+    {
+        ChipLogError(DeviceLayer, "Cannot post event [error: %s]", ErrorStr(error));
```

Comment:
```suggestion
        ChipLogError(DeviceLayer, "Cannot post event: %" CHIP_ERROR_FORMAT, error.Format());
```
We should avoid using `ErrorStr` directly - `CHIP_ERROR_FORMAT` has lower overhead when error strings are disabled.